### PR TITLE
Add groupName attribute to v1 template where it's needed (part 3)

### PR DESCRIPTION
**Pull request by akashorabek**

### CODE REVIEW

File: v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java [hunk 1]
```diff
@@ -277,6 +291,7 @@ public interface Options extends PipelineOptions {
 
     @TemplateParameter.GcsWriteFolder(
         order = 16,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java:294]: Same. This is not the target but an extra output. Leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/BulkDecompressor.java [hunk 2]
```diff
@@ -182,6 +184,7 @@ public interface Options extends PipelineOptions {
 
     @TemplateParameter.GcsWriteFile(
         order = 3,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/BulkDecompressor.java:187]: Same. Not a target, leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java [hunk 3]
```diff
@@ -153,6 +156,7 @@ public interface Options extends DataflowPipelineOptions, CsvPipelineOptions {
 
     @TemplateParameter.GcsWriteFolder(
         order = 4,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java:159]: Same. Not a target, leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java [hunk 4]
```diff
@@ -163,6 +167,7 @@ public interface Options extends DataflowPipelineOptions, CsvPipelineOptions {
 
     @TemplateParameter.BigQueryTable(
         order = 5,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java:170]: Same. Not a target, leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java [hunk 5]
```diff
@@ -312,6 +314,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.Text(
         order = 3,
+        groupName = "Source",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java:317]: I think this is not related to the source and it's a transformation applied in the middle, I'd leave it ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java [hunk 6]
```diff
@@ -326,6 +329,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.Integer(
         order = 4,
+        groupName = "Source",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java:332]: Same. This is about the DLP API call made in the middle. Probably makes sense to combine `inspectTemplateName` and `batchSize` under a separate 3rd group called "DLP Configuration".

File: v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java [hunk 7]
```diff
@@ -349,6 +354,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.ProjectId(
         order = 6,
+        groupName = "Source",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java:357]: Same. Move to "DLP Configuration" group.

File: v1/src/main/java/com/google/cloud/teleport/templates/PubsubToAvro.java [hunk 8]
```diff
@@ -158,6 +163,7 @@ public interface Options
 
     @TemplateParameter.GcsWriteFolder(
         order = 7,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/PubsubToAvro.java:166]: Not a target, leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/PubsubToText.java [hunk 9]
```diff
@@ -128,6 +131,7 @@ public interface Options
 
     @TemplateParameter.GcsWriteFolder(
         order = 4,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/PubsubToText.java:134]: Not a target. Leave ungrouped.

File: v1/src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java [hunk 10]
```diff
@@ -385,6 +385,7 @@ public String apply(String input) {
   public interface TextToBigQueryStreamingOptions extends TextIOToBigQuery.Options {
     @TemplateParameter.BigQueryTable(
         order = 1,
+        groupName = "Target",
```
Comment (an2x) [inline v1/src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java:388]: Not a target. Leave ungrouped.

**Review by an2x — CHANGES_REQUESTED** [review]

File: v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java [hunk 11]
```diff
@@ -132,6 +137,7 @@ public interface Options extends PipelineOptions {
 
     @TemplateParameter.Text(
         order = 6,
+        groupName = "Target",
         description = "ID column",
```
[the remark this answers is not in the document]
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/spanner/ExportPipeline.java [hunk 12]
```diff
@@ -106,6 +109,7 @@ public interface ExportPipelineOptions extends PipelineOptions {
 
     @TemplateParameter.GcsWriteFolder(
         order = 4,
+        groupName = "Target",
```
[the remark this answers is not in the document]
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/spanner/ExportPipeline.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java (hunk 1 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/BulkDecompressor.java (hunk 2 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/BulkDecompressor.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java (hunk 3 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java (hunk 4 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/CSVToBigQuery.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java (hunk 6 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java (hunk 7 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/PubsubToAvro.java (hunk 8 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/PubsubToAvro.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/PubsubToText.java (hunk 9 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/PubsubToText.java]: Done

File: v1/src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java (hunk 10 above)
Comment (Amar3tto) [reply v1/src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java]: Done

**Review by an2x — APPROVED** [review]
